haigha/classes/channel_class.py

```diff
@@ -15,6 +15,12 @@ class ChannelClass(ProtocolClass):
     Implements the AMQP Channel class
     '''
 
+    CLASS_ID = 20
+
+    # Channel method ids for error-recovery code in Channel
+    CLOSE_METHOD_ID = 40
+    CLOSE_OK_METHOD_ID = 41
```

Comment:
Could define all of the 5 channel method ids here, and then use them instead of the "naked" constants in `self. dispatch_map` initialization.